docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -68,6 +68,8 @@ interface State {
 type Props = OwnProps & RouteComponentProps;
 
 class DebugPage extends React.PureComponent<Props, State> {
+  private scrollRef = React.createRef <HTMLDivElement>();
```

Comment:
Question: Is it possible to define a meaningful anchor name to the reference at DOM level, such as `debug-form`?

Reply:
Yes , I'll renaming variable. Thank you for review!

Reply:
Oh, does just renaming the variable change the anchor ID in DOM?